node/tessel-export.js

```diff
@@ -557,19 +562,21 @@ Tessel.Pin.prototype.addListener = function(mode, callback) {
       throw new Error(`Interrupts are not supported on pin ${this.pin}. Pins 2, 5, 6, and 7 on either port support interrupts.`);
     }
 
-    if ((mode === 'high' || mode === 'low') && !callback.listener) {
+    // For one-time 'low' or 'high' event
+    if ((mode === 'low' || mode === 'high') && !callback.listener) {
       throw new Error('Cannot use "on" with level interrupts. You can only use "once".');
     }
 
-    if (this.interruptMode !== mode) {
-      if (this.interruptMode) {
-        throw new Error(`Cannot set pin interrupt mode to ${mode}; already listening for ${this.interruptMode}`);
+    // Can't set multiple listeners when using 'low' or 'high'
+    if (this.interruptMode) {
+      if ((mode === 'low' || mode === 'high') || (this.listeners('low')[0] || this.listeners('high')[0])) {
```

Comment:
That doesn't work either (it doesn't prevent mixing 'high' or 'low' with 'change', 'rise' & 'fall')

This works and might be a little cleaner?
```javascript
if ((mode === 'low' || mode === 'high') || (this.interruptMode === 'low' || this.interruptMode === 'high')) {
```